doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -20,33 +20,35 @@ You need to have _cygwin_ installed and configured if you are running {PRODUCT}
 . After successful installation, add the path to _cygwin_ _bin_ folder to your environment PATH variable. E.g. *C:\cygwin64\bin*
 
 .Procedure
-
-. Log in to {CENTRAL}.
-. Select *Menu* -> *Projects*.
+. In {CENTRAL}, go to *Menu* -> *Projects*.
 . Select or create the space into which you want to import the projects.
 . Click image:project-data/dots.png[] on the right side of the screen and select *Import Project*.
-. In the *Import Project* window, enter the URL (for example, `\https://github.com/USERNAME/REPOSITORY_NAME.git`) and credentials for the Git repository that contains the project that you want to import and click *Import*.
+. In the *Import Project* window, enter the URL (for example, `\https://github.com/USERNAME/REPOSITORY_NAME.git`) and credentials for the GIT repository that contains the project that you want to import and click *Import*.
 +
 The project is added to the {CENTRAL} Git repository and is available in the created or default space.
 +
 [IMPORTANT]
 =====
 Use the HTTPS or Git protocol instead of a SCP-style SSH URL. {CENTRAL} does not support the basic SSH URL and an error appears if you use this URL.
+
+You must have your public ssh key configured in your git provider.
```

Comment:
"publish SSH key ... your Git provider"
"The Git repository ..."
"... KJAR project that contains only a single KJAR and that is compatible ..."

I don't know if you're using the {PROJECT} variable correctly? Is that intended? That's a variable Michele created that renders as "Mortgage_Process". I think you just mean generally with "the project version" right? Either way, that variable probably needs to be made bold or unicode anyway.

Side note: You're a bit all over with git - Git - GIT. I'd search all instances in this doc and make them all "Git" where appropriate (obviously not in file extensions, etc.).

Reply:
Good catch! I don't know why it was {PROJECT}, it should be {PRODUCT}.

Git instances corrected